static/js/hours/open-status/constants.js

```diff
@@ -11,6 +11,6 @@ export const OpenStatusStrings = {
   OPEN_24_HOURS: 'Open 24 Hours',
   CLOSED: 'Closed',
   OPENS_AT: 'Opens at',
-  OPEN_NOW: 'Open now',
+  OPEN_NOW: 'Open Now',
```

Comment:
Will Hitchhikers think we've added a typo if we change this?

Reply:
There was some inconsistency in the theme. From the `open-status-i18n.js` file, the default english text to display is `Open **N**ow`. So I made the msgid as 'Open **N**ow' but the the translations for other languages seems to follow 'Open **n**ow'. I could revert this change and update the .po/.pot files to all use 'Open **n**ow'. This mean the english text would now be `'Open **n**ow'`. (percy will note a bunch of changes in snapshots since english text would be 'Open **n**ow'). What do you think?

Reply:
Ah, yeah let's just not change any behavior! I assumed the current text was "Open now"